foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -152,35 +178,43 @@ HTML comments are not visible to the browser; they allow us to *comment* on our
 In order to write an HTML comment, we just enclose the comment with `<!--` and `-->` tags. For example:
 
 <p class="codepen" data-height="300" data-theme-id="dark" data-default-tab="html,result" data-slug-hash="abwoyBg" data-user="TheOdinProjectExamples" style="height: 300px; box-sizing: border-box; display: flex; align-items: center; justify-content: center; border: 2px solid; margin: 1em 0; padding: 1em;">
+
   <span>See the Pen <a href="https://codepen.io/TheOdinProjectExamples/pen/abwoyBg">
-  html-comments-example</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>)
+  html-comments-example</a> by TheOdinProject (<a href="https://codepen.io/TheOdinProjectExamples">@TheOdinProjectExamples</a>) <!-- markdownlint-disable-line MD044 -->
   on <a href="https://codepen.io">CodePen</a>.</span>
+
 </p>
+
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
 <div class="lesson-note" markdown="1">
 
- **VSCode keyboard shortcut** 
+#### VSCode keyboard shortcut
 
 If you find typing out the comments syntax tiring, the following shortcut will help you quickly create a new comment, convert any line to a comment, or uncomment any line:
 
-- Mac Users: <kbd>Cmd</kbd> + <kbd>/</kbd> 
+- Mac Users: <kbd>Cmd</kbd> + <kbd>/</kbd>
+
 - Windows and Linux Users: <kbd>Ctrl</kbd> + <kbd>/</kbd>
```

Comment:
```suggestion
- Mac Users: <kbd>Cmd</kbd> + <kbd>/</kbd>
- Windows and Linux Users: <kbd>Ctrl</kbd> + <kbd>/</kbd>
```
We don't need new lines between list items, only that the whole list is surrounded by a blank line either side (it was only the issue with the `</div>` tag being directly after the last list item without a blank line in between.